infra/tpu-pytorch-releases/dev_images.tf

```diff
@@ -2,7 +2,7 @@ variable "dev_images" {
   type = list(object({
     accelerator    = string
     arch           = optional(string, "amd64")
-    python_version = optional(string, "3.8")
+    python_version = optional(string, "3.10")
```

Comment:
Do you mind just adding `python_version = "3.10"` to the images you want to update (e.g. [here](https://github.com/pytorch/xla/blob/46e2230012c37ef0cdaf4b9fa7bde1d61e2ebe06/infra/tpu-pytorch-releases/dev_images.auto.tfvars#L9)) instead of updating the default? IMO we should remove this default and explicitly mark the python version we want for each artifact, but that's a separate PR

Reply:
which means I need to discard all the changes in this PR?